source/features/unreleased-commits.tsx

```diff
@@ -62,30 +64,53 @@ export const repoPublishState = new CachedFunction('tag-ahead-by', {
 	cacheKey: cacheByRepo,
 });
 
-async function add(branchSelectorParent: HTMLDetailsElement): Promise<void> {
-	const {latestTag, aheadBy} = await repoPublishState.get();
-	const isAhead = aheadBy > 0;
-
-	if (!latestTag || !isAhead) {
-		return;
-	}
-
+async function createUnreleasedCommitsTag(latestTag: string, aheadBy: number): Promise<HTMLElement> {
 	const commitCount
 		= aheadBy === undeterminableAheadBy
 			? 'more than 20 unreleased commits'
 			: pluralize(aheadBy, '$$ unreleased commit');
-	const label = `There are ${commitCount} since ${latestTag}`;
+	const label = `There are ${commitCount} since ${abbreviateString(latestTag, 30)}`;
 
-	addAfterBranchSelector(
-		branchSelectorParent,
+	return (
 		<a
-			className="btn px-2 tooltipped tooltipped-ne"
+			className="btn px-2 tooltipped tooltipped-se"
```

Comment:
Can you also fix `default-branch-button`?

<img width="256" alt="Screenshot 3" src="https://github.com/refined-github/refined-github/assets/1402241/9c393330-9ca8-4a6a-9d6a-7ce1c99adb46">


Example URL:

https://github.com/refined-github/sandbox/tree/4679-1

Reply:
🤔  I think it is already fixed on https://github.com/refined-github/refined-github/commit/aa9d746180b8d5a96307ae311d5a7adeee73feaa#diff-76f0cf976001e57ff88f7efe4ab593cbcf69929b5ae3e5cc5f085cc85de175e6L33

Reply:
Ah yeah, Safari has an old version